### ISSUE TITLE
Moving keyboard focus while a ToolTip is open shows the new ToolTip only after a delay

### ISSUE THREAD

**Issue by SamBent** [dotnet/wpf#4617]
* .NET Core Version: 3.0+
* Windows version: all
* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? no
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/master/README.md#reporting-security-issues-and-security-bugs).

 
 
 **Problem description:**
1. Run sample app from #4612
3. Press TAB repeatedly to move focus to the TextBox labeled "TextBox"
4. Wait for the ToolTip to appear
5. Press TAB, to move focus to the second Image
 
 **Actual behavior:** 
The Image's ToolTip shows after a delay (5 sec)

 **Expected behavior:**
 ToolTip displays immediately

Moving the mouse to another element shows its ToolTip immediately, if a tooltip is already showing.  Keyboard should work the same way.

**Comment by SamBent** [comment]
Fixed by the tooltip work in 6.0-preview7.
